python/tank/authentication/user.py

```diff
@@ -86,10 +86,7 @@ def refresh_credentials(self):
         Refreshes the credentials of this user so that they don't expire.
         If they are expired, you will be prompted for the user's password.
         """
-        # Make a very simple authenticated request that returns as little information as possible.
-        # If the session token was expired, the ShotgunWrapper returned by create_sg_connection
-        # will take care of the session renewal.
-        self.create_sg_connection().find_one("HumanUser", [])
+        self._impl.refresh_credentials()
```

Comment:
The RV logic doesn't have this method implemented, but it doesn't auto update either, so it won't become an issue on that front. When we release a newer version of the RV integration this is something we need to take into account and have it fixed. Or have RV use ShotgunUser as is and not a custom thing.